types/graphql/utilities/schemaPrinter.d.ts

```diff
@@ -1,8 +1,12 @@
 import { GraphQLSchema } from '../type/schema';
 import { GraphQLType } from '../type/definition';
 
-export function printSchema(schema: GraphQLSchema): string;
+export interface PrinterOptions {
+  [key: string]: any;
```

Comment:
@intellix I was thinking to leave the options open-ended so that when they add another option we don't have to update types again, but I'm fine making them explicit: 2bb2a2f.